Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <LangVersion>7.3</LangVersion>
```

Comment:
I don't think we need to include the `LangVersion` prop. This is a relic from an older preview of Blazor and can be removed.
```suggestion
```

Reply:
you still need it to tell VS what version of c# the code base supports. i have it as 8 in all my repos

Reply:
Only to tie the project to a specific version, there is no need for us to do that. All Blazor template ship without specifying a `LangVersion` prop.

Reply:
how about i change it to `<LangVersion>8.0</LangVersion>` to unlock c# 8 features?

Reply:
In the absence of a `LangVersion` prop the compiler will default the C# lang version based on the target, which in our case is NetStandard2.1 & .NetCore3.x, which is C# 8. So the tag is redundant.

Reply:
so this should be `NetStandard2.1`?  https://github.com/Blazored/Modal/blob/master/src/Blazored.Modal/Blazored.Modal.csproj#L4

Reply:
It will be, I'm await to resolve this PR so I can make some additional updates.

Reply:
ok. langversion removed

Reply:
Great stuff, thanks Simon.